.github/workflows/deploy-to-aks.yaml

```diff
@@ -0,0 +1,108 @@
+name: Create AKS cluster, deploy CKF and run bundle test
+on:
+  workflow_dispatch:
+    inputs:
+      bundle_version:
+        description: 'Comma-separated list of bundle versions e.g. "1.7","1.8"'
+        default: '"1.8"'
+        required: true
+
+  schedule:
+    - cron: "17 0 * * 2"
+jobs:
+  deploy-ckf-to-aks:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        bundle_version: ${{ fromJSON(format('[{0}]', inputs.bundle_version || '"1.7","1.8","latest"')) }}
+      fail-fast: false
+    env:
+      AZURE_CORE_OUTPUT: none
+      K8S_VERSION: ${{ fromJSON('{"1.8":"1.26"}')[matrix.bundle_version] }}
+      JUJU_VERSION: ${{ fromJSON('{"1.8":"3.1"}')[ matrix.bundle_version ] }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install CLI tools tox charmcraft juju
```

Comment:
I would modify the name of this step to just `Install CLI tools`, no need to mention them in the title